Tropos/Controllers/TRAppDelegate.m

```diff
@@ -38,6 +40,32 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     return YES;
 }
 
+- (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken;
+{
+    NSString *channel = [[NSTimeZone localTimeZone] name];
+    [self.courier subscribeToChannel:channel withToken:deviceToken];
+}
+
+- (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo fetchCompletionHandler:(nonnull void (^)(UIBackgroundFetchResult))completionHandler;
+{
+    if (![userInfo[@"aps"][@"content-available"] isEqual: @1]) {
+        completionHandler(UIBackgroundFetchResultNoData);
+        return;
+    }
+    [self.applicationController.conditionsDescription subscribeNext:^(NSAttributedString *conditions) {
```

Comment:
Instead of exposing the `conditionsDescription` property on `TRApplicationController`, what are your thoughts on instead exposing a signal that sends `UILocalNotification`s ready for scheduling? That may simplify the responsibilities of the app delegate, mainly keeping the "view" responsibilities elsewhere.